src/System.Private.ServiceModel/src/System/ServiceModel/Dispatcher/XmlSerializerOperationFormatter.cs

```diff
@@ -568,7 +584,8 @@ internal void SetHeaderAttributes(MessageHeaderDescription headerDescription, bo
                     if (_attributes[headerDescription.Index] == null)
                     {
                         _attributes[headerDescription.Index] = new List<MessageHeader<object>>();
-                    } ((List<MessageHeader<object>>)_attributes[headerDescription.Index]).Add(new MessageHeader<object>(null, mustUnderstand, actor, relay));
+                    } 
+					((List<MessageHeader<object>>)_attributes[headerDescription.Index]).Add(new MessageHeader<object>(null, mustUnderstand, actor, relay));
```

Comment:
Thanks for improving the formatting here. Could you add one more blank line after the closing brace.